app/web/src/components/Customizer/Customizer.tsx

```diff
@@ -28,7 +28,7 @@ const Customizer = () => {
   return (
     <div
       className={`absolute inset-x-0 bottom-0 bg-ch-gray-600 bg-opacity-60 text-ch-gray-300 text-lg font-fira-sans ${
-        open ? 'h-2/3' : ''
+        open ? 'h-1/3' : ''
```

Comment:
I get the gist, but on small screens 1/3 is too small, the point of expanding the parameters for it to expand. how about the following instead?
```suggestion
        open ? 'h-full max-h-96' : ''
```
The other thing I quickly experimented with was using `pointer-events: none` on the upper most div in the customiser panel, this allows navigating around the scene with the orbit/drag controls through the customizer panel, however it also stops scroll events which is a non-starter since the panel needs to be able to scroll.

Reply:
ok, `h-full max-h-96` is much better, I was looking for sth like that, but I do not know tailwind, so I started this PR so we can have a better size definition

Reply:
just noticed , btu ferw seconds too late ... cool feature for colab on PR
![image](https://user-images.githubusercontent.com/2480762/130580893-8cd62821-9ada-450c-97dc-88d350c3bc84.png)